src/applications/personalization/dashboard-2/components/claims-and-appeals/ClaimsAndAppeals.jsx

```diff
@@ -76,11 +77,25 @@ const ClaimsAndAppeals = ({
     return (
       <div>
         <h2>Claims & appeals</h2>
-        <HighlightedClaimAppeal
-          claimOrAppeal={highlightedClaimOrAppeal}
-          name={userFullName}
-        />
-        <ClaimsAndAppealsCTA count={openClaimsOrAppealsCount} />
+        <div className="vads-l-row">
+          <DashboardWidgetWrapper>
+            <HighlightedClaimAppeal
+              claimOrAppeal={highlightedClaimOrAppeal}
+              name={userFullName}
+            />
+            {!highlightedClaimOrAppeal ? (
```

Comment:
placing the CTA in the same `DashboardWidgetWrapper` with the `HighlightedClaimAppeal` makes the layout a single half-width column on large screens if there is no recently updated claim/appeal 

<img width="1074" alt="old over 1008w" src="https://user-images.githubusercontent.com/20728956/113633320-96061400-9621-11eb-8833-075f35a894b2.png">